app/forms/provider_interface/rejection_reasons_wizard.rb

```diff
@@ -2,12 +2,18 @@ module ProviderInterface
   class RejectionReasonsWizard
```

Comment:
should we rename this to `rejections` now we changed the routes 🤔

Reply:
Yeah good idea, this will help distinguish it from the legacy wizard. Will amend.